keps/sig-autoscaling/20181106-in-place-update-of-pod-resources.md

```diff
@@ -0,0 +1,365 @@
+---
+title: In-place Update of Pod Resources
+authors:
+  - "@kgolab"
+  - "@bskiba"
+  - "@schylek"
+  - "@vinaykul"
+owning-sig: sig-autoscaling
+participating-sigs:
+  - sig-node
+  - sig-scheduling
+reviewers:
+  - "@bsalamat"
+  - "@derekwaynecarr"
+  - "@dchen1107"
+approvers:
+  - TBD
```

Comment:
Please identify the component owners (for the autoscaling/node/scheduling areas) that will approve this KEP. That helps ensure there's [agreement on the goals and overall approach before entering the API review process](https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md#mechanics).

Reply:
@liggitt Thanks for pointing this out. I've identified the approvers for the stakeholder SIGs, and SIG-node, SIG-scheduling have approved the KEP. 
@mwielgus is going to follow-up with @kgolab to see if there are any concerns, and if not we should get lgtm and approval from SIG-autoscaling. 

Please let us know what our next-steps are for API review.
Thanks,

Reply:
> @liggitt Thanks for pointing this out. I've identified the approvers for the stakeholder SIGs, and SIG-node, SIG-scheduling have approved the KEP.
> @mwielgus is going to follow-up with @kgolab to see if there are any concerns, and if not we should get lgtm and approval from SIG-autoscaling.
> 
> Please let us know what our next-steps are for API review.

Thanks. I'd suggest:
1) merging this in provisional state
2) capturing links to outstanding comments/threads in an issue for resolution in a follow-up PR
3) reaching out to the API approver (it looks like @thockin self assigned this one) to schedule a time for review

Reply:
@liggitt Thanks for the guidance. I've resolved many of the issues and comments that were either addressed or have become stale.

I'm tracking the remaining outstanding questions in https://github.com/kubernetes/enhancements/issues/1287 

I'll give folks a few days to re-open any that they may feel is not resolved or resolved in error.
And then I and @dashpole will ping @thockin to setup a time for API review.